index.html

```diff
@@ -10,19 +10,23 @@
         <link rel="stylesheet" href="normalize.css">
         <link rel="stylesheet" href="skeleton.css">
         <link rel="stylesheet" href="styles.css"/>
+        <link rel="stylesheet" href="icons/icon.css">
 
         <script src="js/modernizr.js"></script>
 
         <link rel="icon" type="image/png" href="imgs/favicon.png">
 
+        <style>
+
+        </style>
+
     </head>
     <body>
         <!--[if lt IE 8]>
             <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
 
-        <header>
-          <div class="container">
+        <header class="pageHeader container">
```

Comment:
I may have forgotten to mention this, but hyphen-spacing is the convention for classes and camelCasing is the convention for Ids. Not a big deal now, but it can help further separate concerns later with JS Interaction